lib/presentation/view/custom_list_tiles/shared_space_member_list_tile_builder.dart

```diff
@@ -29,68 +29,92 @@
+      trailing: SharedSpaceOperationRole.deleteMemberSharedSpaceRoles.contains(userCurrentRole)
+        ? GestureDetector(
+          onTap: () {
+            if (onDeleteMemberCallback != null) {
+              onDeleteMemberCallback.call();
+            }
+          },
+          child: Icon(Icons.close, color: AppColor.deleteMemberIconColor))
+        : SizedBox.shrink(),
       title: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
         Padding(
-            padding: EdgeInsets.only(bottom: 8),
-            child: Text(_name,
-                style: TextStyle(
-                    fontSize: 14,
-                    fontWeight: FontWeight.w500,
-                    fontStyle: FontStyle.normal,
-                    color: AppColor.loginTextFieldTextColor))),
-        Text(_email,
+          padding: EdgeInsets.only(bottom: 8),
+          child: Text(_name,
             style: TextStyle(
-                fontSize: 14,
-                fontWeight: FontWeight.normal,
-                fontStyle: FontStyle.italic,
-                color: AppColor.documentModifiedDateItemTextColor)),
+              fontSize: 14,
+              fontWeight: FontWeight.w500,
+              fontStyle: FontStyle.normal,
+              color: AppColor.loginTextFieldTextColor))),
+        Text(_email,
+          style: TextStyle(
+            fontSize: 14,
+            fontWeight: FontWeight.normal,
+            fontStyle: FontStyle.italic,
+            color: AppColor.documentModifiedDateItemTextColor)),
         Padding(
-            padding: EdgeInsets.only(top: 8),
-            child: GestureDetector(
-              onTap: () {
-                if(onSelectedRoleCallback != null) {
-                  onSelectedRoleCallback.call();
-                }
-              },
-              child: Row(
-                mainAxisSize: MainAxisSize.min,
-                children: [
-                  Text(_roleName,
-                      style: TextStyle(
-                          fontSize: 14,
-                          fontWeight: FontWeight.normal,
-                          fontStyle: FontStyle.normal,
-                          color: AppColor.workgroupNodesSurfingBackTitleColor)),
-                  Icon(Icons.arrow_drop_down, color: AppColor.primaryColor)
+          padding: EdgeInsets.only(top: 8),
+          child: GestureDetector(
+            onTap: () => _handleUpdateRoleTap(),
+            child: Row(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(_roleName,
+                  style: TextStyle(
+                    fontSize: 14,
+                    fontWeight: FontWeight.normal,
+                    fontStyle: FontStyle.normal,
+                    color: AppColor.workgroupNodesSurfingBackTitleColor)),
+                SharedSpaceOperationRole.editMemberSharedSpaceRoles.contains(userCurrentRole)
+                ? Icon(Icons.arrow_drop_down,
+                  color: AppColor.primaryColor)
```

Comment:
`color` is a parameter of `Icon` constructor. It can not be the same indent with `Icon()`